### ISSUE TITLE
API for deleting an account

### ISSUE THREAD

**Issue by deiu** [linkeddata/gold#28]

**Comment by dmitrizagidulin** [comment]
Do we need to open a similar feature request on https://github.com/linkeddata/ldnode/ ?

(In general, should Solid API feature requests like this one be tracked on https://github.com/solid/solid-spec ?)

**Comment by nicola** [comment]
@dmitrizagidulin yes, that is correct. I think about a year ago there was no repo github.com/solid, hence everything started in here (this is my take!)
